src/main/java/lotto/LottoGame.java

```diff
@@ -0,0 +1,45 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class LottoGame {
+    private final static int LOTTO_PRICE = 1000;
+    private List<Lotto> lottos = new ArrayList<>();
+
+    public LottoGame(int lottoCount, LottoGeneration lottoGeneration) {
+        Stream.generate(() -> Lotto.create(lottoGeneration)).limit(lottoCount)
+                .forEach(item -> lottos.add(item));
+    }
+
+    public static int LOTTO_PRICE() {
```

Comment:
메서드 네이밍 컨벤션에 맞춰주시면 좋을 것 같네요. 상수를 반환한다고 메서드도 대문자로 네이밍하진 않습니다.

```suggestion
    public static int getLottoPrice() {
```